sbin/prepareWorkspace.sh

```diff
@@ -165,6 +171,19 @@ checkoutRequiredCodeToBuild() {
       else
         echo "Failed cmd: git fetch origin \"refs/tags/${tag}:refs/tags/${tag}\""
       fi
+    elif [ "$sha" ]; then
+      echo "Checking out sha ${sha}"
+      git checkout "${sha}" || rc=$?
+      if [ $rc -eq 0 ]; then
```

Comment:
Any reason not to use `if git checkout "${sha}"` here? The use of `$rc` here look a little complicated.

Reply:
Just to keep the same style as other checks.